server/src/test/java/org/apache/iotdb/db/metadata/idtable/IDTableRecoverTest.java

```diff
@@ -63,32 +63,32 @@ public void before() {
     isEnableIDTableLogFile = IoTDBDescriptor.getInstance().getConfig().isEnableIDTableLogFile();
 
     IoTDBDescriptor.getInstance().getConfig().setEnableIDTable(true);
-    IoTDBDescriptor.getInstance().getConfig().setDeviceIDTransformationMethod("SHA256");
+    IoTDBDescriptor.getInstance().getConfig().setDeviceIDTransformationMethod("AutoIncrement");
```

Comment:
Why change this? If you want to test the ```AutoIncrement``` mode, it's better to add new tests. If you want to reuse the test case of ```SHA256```, maybe test class extension will work.